src/partials/languages/el.html

```diff
@@ -34,9 +34,7 @@
   "meta.comments": "Σχόλια",
   "meta.source": "Πηγή",
   "nav.title": "Πλοήγηση",
-  "search.config.lang": "el",
-  "search.config.pipeline": "trimmer, stopWordFilter",
-  "search.config.separator": "[\s\-]+",
+  "search.config.pipeline": " ",
```

Comment:
I think the `trimmer` and `stopWordFilter` should not be removed. My testing showed it also works with them.